.env.example

```diff
@@ -1,6 +0,0 @@
-AWS_ACCESS_KEY_ID=""
-AWS_SECRET_ACCESS_KEY=""
-S3_BUCKET=""
-
-MANDRILL_USERNAME=""
-MANDRILL_APIKEY=""
```

Comment:
It became .env instead of .env.example between the last time I pulled from master and updating the branch